Jenkinsfile-sbom-release

```diff
@@ -0,0 +1,194 @@
+      withSonatypeDockerRegistry() {
+        sh "docker pull sonatype/nexus3:${params.docker_nexus3_tag}"
+        env['nexusVersion'] = sh(script: "docker inspect sonatype/nexus3:${params.docker_nexus3_tag} \
+            | jq -r '.[0].Config.Labels.version' ",
+          returnStdout: true).trim()
+        env['dockerImageVersion'] = sh(script: "docker inspect sonatype/nexus3:${params.docker_nexus3_tag} \
+            | jq -r '.[0].Config.Labels.release' ",
+          returnStdout: true).trim()
+        env['ubiImageId'] = sh(script: "docker inspect sonatype/nexus3:${params.docker_nexus3_tag} \
+            | jq -r '.[0].Config.Labels.\"base-image-ref\"' \
+            | sed -En 's/^.+image=(.+)\$/\\1/p'",
+          returnStdout: true).trim()
+      }
+    },
+    run: {
+      def buildDir = "./.sbom-build/job-${env.BUILD_NUMBER}"
+      def ubiImageName = sh(script: "curl -s -X 'GET' '${REDHAT_CONTAINER_API_URL_BASE}/images/id/${env.ubiImageId}' -H 'accept: application/json' \
+          | jq -r '.brew.build' \
+          | sed -En 's/(ubi[0-9]+-minimal)-container-([0-9]+\\.[0-9]+-[0-9]+\\.?[0-9]*)/\\1-\\2/p'",
+          returnStdout: true).trim()
+      def ubiImageVersion = sh(script: "curl -s -X 'GET' '${REDHAT_CONTAINER_API_URL_BASE}/images/id/${env.ubiImageId}' -H 'accept: application/json' \
+          | jq -r '.brew.build' \
+          | sed -En 's/ubi[0-9]+-minimal-container-([0-9]+\\.[0-9]+-[0-9]+\\.?[0-9]*)/\\1/p'",
+          returnStdout: true).trim()
+
+      // Download SBOMs
+      sh "mkdir -p ${buildDir}/spdx && mkdir -p ${buildDir}/cyclonedx"
+
+      // Get nexus-internal SBOM
+      getComponentSbom(buildDir, "nexus-internal", env.nexusVersion)
+      // Get nxrm-db-migrator SBOM
+      getComponentSbom(buildDir, "nxrm-db-migrator", env.nexusVersion)
+      // Get docker-nexus3 SBOM
+      getComponentSbom(buildDir, "docker-nexus3", env.dockerImageVersion)
+      // Get UBI Minimal SBOM
+      def ubiSbomAvailable = getUbiImageSbom(buildDir, ubiImageName, ubiImageVersion)
+
+      sh "echo 'Available SPDX SBOMS' && ls ${buildDir}/spdx"
+      sh "echo 'Available CycloneDx SBOMS' && ls ${buildDir}/cyclonedx"
+
+      // Merge supported sboms
+      def dockerImageNamespace = sh(script: "cat ${buildDir}/spdx/docker-nexus3-${env.dockerImageVersion}-spdx.json | jq -r '.documentNamespace'", returnStdout: true).trim()
+      mergeSpdxComponents(buildDir, "docker-nexus3-aggregate", env.dockerImageVersion, dockerImageNamespace)
+
+      // Publish SBOMs
+      if (ubiSbomAvailable) {
+        publishComponent(buildDir, "ubi-minimal", ubiImageVersion, false)
+      }
+      publishComponent(buildDir, "nexus-internal", env.nexusVersion)
+      publishComponent(buildDir, "nxrm-db-migrator", env.nexusVersion)
+      publishComponent(buildDir, "docker-nexus3", env.dockerImageVersion)
+      publishComponent(buildDir, "docker-nexus3-aggregate", env.dockerImageVersion, false)
+
+      sh "rm -rf '${buildDir}'"
+    }
+)
```

Comment:
Nit: new line missing.

Reply:
Resolved in [ee38b2e](https://github.com/sonatype/docker-nexus3/pull/191/commits/ee38b2e875b7b008d5a718f4a54727534e660e56)